students/Net_Michael/lesson3/list_lab.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# series 1
+# Create and display the given list 
+
+fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruit)
+
+
+# In[2]:
+
+
+# series 1
+# Ask the user for another fruit add it to the end of the list
+# display the list
+
+new_fruit = input("Please add new fruit to the list ")
+output = fruit.append(new_fruit)
+print(fruit)
+
+
+# In[3]:
+
+
+# series 1
+# Ask the user for a number  display the number back to the user 
+
+def sel_fruit(f, fruit = fruit):
+    m = len(fruit) + 1
+    while f <= m:
+        for i in range(1,m):
+            if f == i : 
+                sel_fr = fruit[i-1]
+                return(sel_fr)
+
+index = input("Fruits indexed by ")
+print("are " + sel_fruit(int(index)) + ".")
+
+
+# In[4]:
+
+
+# series 1
+# add another fruit to the list using +
+
+new_fruit = input("Add new fruit to the list ")
+output = [new_fruit] + fruit
+print( output)
+
+
+# In[5]:
+
+
+# series 1
+# add new fruit at the begining of the list using insert()
+
+add_fruit = input("Add new fruit at the begining of the list ")
+output = fruit.insert(0, add_fruit)
+print(fruit) 
+
+
+# In[6]:
+
+
+# series 1
+# display all the fruits that begin with "P" 
+            
+def first_letter(fruit, word_a):
+    m = len(fruit)
+    for i in range(m):
+        if word_a in fruit[i]:
```

Comment:
since you only want fruits that start with a P you should either check via `[0]` index or `startswith` function:
```
for fruit in fruits:
    if fruit[0] == 'P':
        return True
```